### PR TITLE
CRO 0.32.1 release preparation

### PR DESCRIPTION
**Pull request by makslion**
## Overview
Preparing release

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,13 +4,13 @@ IMAGE_NAME ?= cloud-resource-operator
...
-PREV_VERSION=0.31.0
-VERSION=0.32.0
+PREV_VERSION=0.32.0
+VERSION=0.32.1
 COMPILE_TARGET=./tmp/_output/bin/$(IMAGE_NAME)
 UPGRADE ?= true
 CHANNEL ?= rhmi
 
-PREVIOUS_OPERATOR_VERSIONS="0.31.0,0.30.0,0.29.0,0.28.0,0.27.1,0.27.0,0.26.0,0.25.0,0.24.1,0.24.0,0.23.0"
+PREVIOUS_OPERATOR_VERSIONS="0.32.1,0.31.0,0.30.0,0.29.0,0.28.0,0.27.1,0.27.0,0.26.0,0.25.0,0.24.1,0.24.0,0.23.0"
```
Comment (MStokluska) [inline Makefile:13]: @makslion @laurafitzgerald I believe this is a typo? 0.32.1 should not be in previous versions but 0.32.0 should.

Comment (makslion) [reply Makefile:13]: yep, it should be 0.32.0 (if I'm following version correctly)

**Comment by MStokluska** [comment]
@makslion can we get version.go Version updated accordingly as well please? Issue is explained in this pr https://github.com/integr8ly/cloud-resource-operator/pull/242

**Pull request merged**